### ISSUE TITLE
debounce's cancel() fn missing when webpacked

### ISSUE THREAD

**Issue by binarykitchen** [jashkenas/underscore#2719]
I am not 100% sure if that's really the case. But when I look at the bundle webpack has generated, it seems it did remove the `.cancel()` function debounce is supposed to return with the debounced instance.

This here is the code from webpack. There is no cancel():

```js
  _.debounce = function(func, wait, immediate) {
    var timeout, args, context, timestamp, result;

    var later = function() {
      var last = _.now() - timestamp;

      if (last < wait && last >= 0) {
        timeout = setTimeout(later, wait - last);
      } else {
        timeout = null;
        if (!immediate) {
          result = func.apply(context, args);
          if (!timeout) context = args = null;
        }
      }
    };

    return function() {
      context = this;
      args = arguments;
      timestamp = _.now();
      var callNow = immediate && !timeout;
      if (!timeout) timeout = setTimeout(later, wait);
      if (callNow) {
        result = func.apply(context, args);
        context = args = null;
      }

      return result;
    };
  };
```

Using latest underscorejs version 1.83 here.

**Comment by captbaritone** [comment]
Can you reproduce this with a project which consists of only Underscore and has a minimal config?

**Comment by andresroberto** [comment]
That's because the .cancel function inside the debounce implementation was added on November 5th 2015 as .clear ([see commit](https://github.com/jashkenas/underscore/commit/c7bb6359a4fd57113b5a6f4ac9e89012c179e87a#diff-0f36b362a0b81d6f4d4bfd8a7413c75d)) and renamed to .cancel on November 10th 2015 ([see commit](https://github.com/jashkenas/underscore/commit/e8c252b205813fbb4070cf87118f6502dc1099be#diff-0f36b362a0b81d6f4d4bfd8a7413c75d)) both of which happened after the latest release of Undercore (1.8.3) which took place on April 2nd 2015.

So, that implementation change is actually not inside the version you are using. In order to show that I created a project which consists of only underscore being imported in a js file and webpack with no config bundling it. I've created two branches, [master](https://github.com/andresroberto/webpack-underscore) which makes use of version 1.8.3 of underscore and does not include the .cancel function, and [underscore-master](https://github.com/andresroberto/webpack-underscore/tree/underscore-master) which pulls underscore from the master branch of the repo and it does contain the .cancel function both in the source and in the bundle.
